circle.yml

```diff
@@ -30,12 +30,20 @@ stages:
         command: |
           set -e
           echo $HEROKU_DEB_KEY | base64 -d | gpg --import
+          cat << EOF > ~/.netrc
+          machine api.heroku.com
+            login jeff+devcenter@heroku.com 
```

Comment:
You may recall I actually did a screencast of that not working on my local machine and posted it in slack. Suffice to say I had some trouble with it.  It's working CI now. I don't know what changed, but it wasn't working before.